Woof/WoofTests/Model/LocalStorage/LocalStorageDeleteTests.swift

```diff
@@ -0,0 +1,48 @@
+import XCTest
+
+final class LocalStorageDeleteTests: XCTestCase {
+    private var localStorage: LocalStorage!
+    private let defaultValueToSave = "Hello, World!".data(using: .utf8)!
+
+    override func setUp() {
+        let storage = LocalStorage()
+        localStorage = storage
```

Comment:
```suggestion
        localStorage = LocalStorage()
```
?

What are you trying to achieve here?

Reply:
fixed